### PR TITLE
Feature: Bookmarks Flow

### PR DESCRIPTION
**Pull request by itsnaidu**
https://user-images.githubusercontent.com/47970992/181417152-0b892dc3-8807-440b-88b5-65e2490d1b88.mov

### CODE REVIEW

**Comment by itsnaidu** [comment]
https://user-images.githubusercontent.com/47970992/182373830-a7ab7feb-618f-4804-ba2e-d71b1ac7367d.mov

File: ViewControllers/MoreItemsViewController.swift [hunk 1]
```diff
@@ -51,7 +51,7 @@ class MoreItemsViewController: UIViewController {
     }
     
     func showAlertToLogoutConfirmation() {
-        let logoutConfirmationAlert = UIAlertController(title: "Confirm Logout", message: "Are you sure you wish to logout?", preferredStyle: .alert)
+        let logoutConfirmationAlert = UIAlertController(title: "Confirm Logout", message: "Please confirm you are logging out", preferredStyle: .alert)
```
Comment (RabidFire) [inline ViewControllers/MoreItemsViewController.swift]: Revert

File: Utils/ViewController+DashX.swift [hunk 2]
```diff
@@ -46,8 +46,8 @@ extension UIViewController {
         }
     }
     
-    func showError(with message: String) {
-        let errorAlertVC = UIAlertController(title: "Error", message: message, preferredStyle: .alert)
+    func showError(with message: String?) {
```
Comment (RabidFire) [inline Utils/ViewController+DashX.swift:49]: Rename file

File: SceneDelegate.swift [hunk 3]
```diff
@@ -6,6 +6,7 @@
 //
 
 import UIKit
+import DashX
```
Comment (RabidFire) [inline SceneDelegate.swift]: Remove unnecessary import

**Pull request merged**